packages/input_schema/package.json

```
@@ -36,8 +36,10 @@
     },
     "dependencies": {
         "@apify/consts": "^1.2.0",
-        "ajv": "^6.12.6",
         "countries-list": "^2.6.1",
         "escaya": "^0.0.61"
+    },
+    "peerDependencies": {
+        "ajv": "^8.6.2"
```

peer deps are usually just about major versions, so as long as we know there are no major issues with 8.0.0, i'd suggest to use that instead

```suggestion
        "ajv": "^8.0.0"
```